### PR TITLE
Initial test script

### PR DESCRIPTION
**Pull request by dnahodil**

### CODE REVIEW

File: test_jodaac.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
```
Comment (danfruehauf) [inline test_jodaac.sh]: Remove the space here.

File: test_jodaac.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+NUM_JOBS=${1:-4}
+NUM_CONCURRENT_JOBS=${2:-$NUM_JOBS}
+
+JODAAC_HOST="http://localhost:8080/netcdf-subset"
+JODAAC_URL="$JODAAC_HOST/subset?typeName=anmn_ts&CQL_FILTER=INTERSECTS(geom,POLYGON((113.33 -33.09,113.33 -30.98,117.11 -30.98,117.11 -33.09,113.33 -33.09))) AND TIME >= %272015-01-13T23:00:00Z%27 AND TIME <= %272015-04-14T00:00:00Z%27"
+JODAAC_COMMAND="wget '$JODAAC_URL' -O 'download{#}.zip'"
+
+echo "Starting $NUM_JOBS jobs..."
+seq "$NUM_JOBS" | parallel -j"$NUM_CONCURRENT_JOBS" $JODAAC_COMMAND
```
Comment (danfruehauf) [inline test_jodaac.sh:12]: Just so you know, `parallel` will work only on Linux and only on some distributions. It's not considered particularly portable...

**Comment by danfruehauf** [comment]
I personally thought you might take a JMeter approach, but bash does the trick too.

**Comment by dnahodil** [comment]
I was working from a simple test script @julian1 had knocked up. If I knew how much I'd end up changing it I might have gone down a different route. It's not a lot of work to lose if we decide to go a different way.

**Comment by dnahodil** [comment]
These damn tests

**Comment by danfruehauf** [comment]
Changes are good. Just sort the tests.

**Pull request merged**